django/db/backends/sqlite3/schema.py

```diff
@@ -143,19 +142,17 @@ def is_self_referential(f):
 
         # Work out the new value of unique_together, taking renames into
         # account
-        if override_uniques is None:
-            override_uniques = [
-                [rename_mapping.get(n, n) for n in unique]
-                for unique in model._meta.unique_together
-            ]
+        override_uniques = [
```

Comment:
How about naming these variables `unique_together` / `index_together` rather than the "override" prefix?